driver/src/main/com/mongodb/reactivestreams/client/internal/GridFSAsyncStreamHelper.java

```diff
@@ -45,42 +43,43 @@
      * @param wrapper the callback AsyncInputStream
      * @return the Publisher AsyncInputStream
      */
-    public static AsyncInputStream toAsyncInputStream(final com.mongodb.async.client.gridfs.AsyncInputStream wrapper) {
```

Comment:
Sigh - the change here is to use fully qualified names now AsyncInput/OutputStream are deprecated.